openupgrade_scripts/scripts/website_sale/17.0.1.1/post-migration.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2025 Tecnativa - Pilar Vargas
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    # Set to False so as not to change the behaviour of the website with new things.
+    env.cr.execute(
```

Comment:
this should be `logged_query`, but let's do it in website_sale_digital migration

Reply:
It has given time